# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/doc/Debugger/Debugger.Frame.md
# Commit: d70ec1e79887
# Full Hash: d70ec1e7988710de0b79351f730a2498972cd7e0
# Author: Logan Smyth <loganfsmyth@gmail.com>
# Date: 2019-12-09 09:50:39
# Regressor Bug: 1600204
# File Overlap Count: 1
# Description:
#   Bug 1600204 - Part 1: Support .type on suspended generator frames. r=jimb
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D55231
# ==============================================================================

diff -r e3354d109039 -r d70ec1e79887 js/src/doc/Debugger/Debugger.Frame.md
--- a/js/src/doc/Debugger/Debugger.Frame.md	Fri Dec 06 21:44:23 2019 +0000
+++ b/js/src/doc/Debugger/Debugger.Frame.md	Fri Dec 06 02:29:18 2019 +0000
@@ -181,7 +181,7 @@
 * `"debugger"`: a frame for a call to user code invoked by the debugger
   (see the `eval` method below).
 
-Accessing this property will throw if `.onStack == false`.
+Accessing this property will throw if `.terminated == true`.
 
 ### `implementation`
 A string describing which tier of the JavaScript engine this frame is